package.json

```diff
@@ -7,19 +7,19 @@
     "dev": "vite --port 3000",
     "dev4container": "vite --port 3000 --host 0.0.0.0",
     "start": "vite --port 3000 --host 0.0.0.0",
-    "build": "vite build",
+    "build": "tsc && vite build",
     "lint": "eslint . --ext js,jsx --report-unused-disable-directives --max-warnings 0",
     "preview": "vite preview --host 0.0.0.0 --port 3000",
     "serve": "serve -s dist",
     "generate_svg_to_component": "npx @svgr/cli --out-dir src/images/icons/components -- src/images/icons",
-    "test": ""
+    "format": "prettier --write ./src",
```

Comment:
czy wszystkie pliki są już po formacie prettiera?

Reply:
No, I do not launch it. Tommorow I will do it.